unit_tests/src/main.cpp

```diff
@@ -13,7 +13,7 @@ int main( int argc, char* argv[] )
   // this is the preferred way to set them
 
   // Build a new parser on top of Catch's
-  using namespace Catch::clara;
+  using namespace Catch::Clara;
```

Comment:
noting that Clara appears unsupported/archived now... https://github.com/catchorg/Clara

Points to fork at https://github.com/bfgroup/Lyra

Reply:
carry on.